Android/app/src/main/java/app/intra/sys/IntraVpnService.java

```diff
@@ -60,6 +58,9 @@
   private static final String WARNING_CHANNEL_ID = "warning";
   private static final String NO_PENDING_CONNECTION = "This value is not a possible URL.";
 
+  // Factory for producing DoH connections.
+  private final ServerConnectionFactory dohFactory = new ServerConnectionFactory(this);
```

Comment:
Do you need this to be an instance variable? I only see it used in `updateServerConnection`.

Reply:
Done